BUILD.gn

```diff
@@ -333,7 +333,7 @@ if (target_cpu == "x86") {
 
 action("create_symbols_dist") {
   if (is_android) {
-    output = "$brave_dist_dir/$brave_product_name-v$brave_version-$brave_platform-$target_android_base-$target_cpu-symbols.zip"
+    output = "$brave_dist_dir/$brave_product_name-v$brave_version-$brave_platform-$target_android_output_format-$target_android_base-$target_cpu-symbols.zip"
```

Comment:
shouldn't this be moved more to the end?

Reply:
Do you mean to be
`"$brave_dist_dir/$brave_product_name-v$brave_version-$brave_platform-$target_android_base-$target_cpu-symbols-$target_android_output_format.zip"`


so for example instead of  `brave-v1.31.23-android-aab-mono-arm64-symbols.zip`
to be `brave-v1.31.23-android-mono-arm64-symbols-aab.zip` ?

I can change, this is not so important.

Reply:
yes, like that

Reply:
This is fixed in https://github.com/brave/brave-core/pull/9991/commits/dfc940e89c894f11a3d3755360343ed4db35d98c